install-prereqs.sh

```diff
@@ -33,7 +33,7 @@ if yum --help; then
   yum install -y gettext
 
 elif apt --help; then
-  apt-get install -y --no-install-recommends build-essential cmake libssl-dev zlib1g-dev libffi-dev
+  apt-get install -y --no-install-recommends build-essential cmake libssl-dev zlib1g-dev libffi-dev uuid-runtime
```

Comment:
Needed for `uuidgen` used here
https://github.com/scylladb/scylla-cluster-tests/blob/b6391f90df2fb7807646fd015031ac092ce3fd13/docker/env/hydra.sh#L41